prime-router/metadata/hl7_mapping/common/datatype/hd-location.yml

```diff
@@ -0,0 +1,13 @@
+elements:
```

Comment:
This file is potentially a starting point for having future schemas broken down by a combination of FHIR resource and HL7V2 datatype. This makes things a bit easier to understand than having one file cover both Location->HD and Location->PL, or both Location->HD and ExtensionHierarchicDesignator->HD.

Reply:
What do you think about renaming the file or maybe even a different directory structure?  Long term it might be lost that is specifically splitting up mapping one FHIR resource to two HL7 ones